package/kernel/mac80211/files/lib/wifi/mac80211.sh

```diff
@@ -77,9 +93,19 @@ detect_mac80211() {
 		[ "$found" -gt 0 ] && continue
 
 		mode_band="g"
-		channel="11"
+		channel="1"
 		htmode=""
 		ht_capab=""
+		encryption="psk2"
+		key="$secure_psk"
+		[ -n "$wifi_txpower" ] && tx_power="set wireless.radio${devidx}.txpower=$wifi_txpower" \
+			|| [ -n "$wifi_country" ] || tx_power="set wireless.radio${devidx}.txpower=5"
+		[ -n "$wifi_country" ] && country="$wifi_country" || country="XX"
```

Comment:
Would it make sense to set the country to 00/international?

Reply:
I don't know XX. I only know that 00 is international and thus also most restrictive.

Reply:
[It seems](https://git.kernel.org/pub/scm/linux/kernel/git/sforshee/wireless-regdb.git/tree/db.txt) there is no XX